lib/frontend/src/actions/meetings.js

```diff
@@ -203,12 +203,12 @@ export const submitOrUpdateReport = (
   let response, json, method, url
   const state = getState()
   const newReport = state.meeting.results
-  for (let i = 0; i < newReport.indicators.length; i++) {
-    newReport.indicators[i].value =
-      newReport.indicators[i].value === ''
-        ? null
-        : parseFloat(newReport.indicators[i].value)
-  }
+  newReport.indicators.map(
```

Comment:
Vaut mieux utiliser un forEach qu'un map quand tu n'as pas besoin de la valeur de retour.

Reply:
c'est noté, merci pour la revue de code :)